bot_python_sdk/store.py

```diff
@@ -1,76 +1,107 @@
 import json
 import os
+from bot_python_sdk.logger import Logger
 
 _actions_file_path = 'storage/actions.json'
 _last_triggered_file_path = 'storage/last_triggered.json'
 _qr_image_path = 'storage/qr.png'
 _configuration_file_path = 'storage/configuration.json'
 _bot_public_key = 'storage/public.pem'
 
+LOCATION = 'Store'
+
 
 class Store:
+    def __init__(self):
+        pass
+
     @staticmethod
     def set_actions(actions):
-        file = open(_actions_file_path, 'w')
-        file.write(json.dumps(actions))
-        file.close()
+        try:
+            with open(_actions_file_path, 'w') as file:
+                file.write(json.dumps(actions))
+        except IOError as io_error:
+            Logger.error(LOCATION, 'Unable to write in {0} path'.format(_actions_file_path))
```

Comment:
could we just log the `io_error.message` here as the second argument?

Reply:
error message has been added as second argument in Log

Reply:
I think its best to just log the io_error.message and not add the `unable to write`
What is in the io_error message? Do you have an example?

Reply:
yes, after adding the error message the both will mean the same.  So i have removed the string and logged only the error message. PFB for the sample error message 
No such file or directory: 'path'